luiz/ba2b.py

```diff
@@ -0,0 +1,42 @@
+# /usr/bin/env python
```

Comment:
should this be #! or am I missing out on some black magic?

Reply:
:see_no_evil: oops, fixed!